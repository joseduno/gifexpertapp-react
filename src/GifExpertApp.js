import React, { useState } from 'react'
import AddCategory from './components/AddCategory'
import GifGrid from './components/GifGrid'

const GifExpertApp = () => {
    
    const [category, setCategory] = useState('King Kong')

    return (
        <>
            <h2 className='animate__animated animate__fadeInDownBig'>GifExpertApp</h2>
            <hr/>
            <br />
            <AddCategory setCategory={setCategory}/>
            <GifGrid category={category} />

        </>
    )
}

export default GifExpertApp
