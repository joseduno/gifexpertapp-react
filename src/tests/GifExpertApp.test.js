import { shallow } from 'enzyme'
import GifExpertApp from '../GifExpertApp'


describe('Pruebas en el componente GifExpertApp', () => {

    test('debe mostrar correctamente el componente', () => {
        const wrapper = shallow(<GifExpertApp />)
        expect(wrapper).toMatchSnapshot()
    })
    
})
