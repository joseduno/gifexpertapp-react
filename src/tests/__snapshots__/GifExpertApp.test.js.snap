// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componente GifExpertApp debe mostrar correctamente el componente 1`] = `
<Fragment>
  <h2
    className="animate__animated animate__fadeInDownBig"
  >
    GifExpertApp
  </h2>
  <hr />
  <br />
  <AddCategory
    setCategory={[Function]}
  />
  <GifGrid
    category="King Kong"
  />
</Fragment>
`;
